xonsh/completers/ssh.py

```diff
@@ -0,0 +1,39 @@
+
+import os
+import subprocess
+
+def get_host_list():
+    '''
+    Reads ~/.ssh/known_hosts and returns a list of hosts
+    '''
+    hosts_path = os.path.expanduser('~/.ssh/known_hosts')
+    if not os.path.exists(hosts_path):
+        return []
+    with open(hosts_path, 'r') as hosts_file:
+        host_list = []
+        for line in hosts_file:
+            line = line.split(' ')[0]
+            if ',' in line:
+                line = line.split(',')
+                for item in line:
+                    host_list.append(item.split(':')[0].strip('[').strip(']'))
+            elif ':' in line:
+                host_list.append(line.split(':')[0].strip('[').strip(']'))
+            else:
+                host_list.append(line)
+        return host_list
+
+
+def complete_ssh(prefix, line, begidx, endidx, ctx):
+    '''
+    Completes ssh hosts based on contents of ~/.ssh/known_hosts
```

Comment:
Nice to see ssh completer coming!

Should the hosts be retrieved from `.ssh/config` file too?